build.gradle

```diff
@@ -67,8 +67,12 @@ buildscript {
               'js': "org.jetbrains.kotlin:kotlin-test-js",
           ],
           'coroutines': [
-              'core': "org.jetbrains.kotlinx:kotlinx-coroutines-core:${versions.coroutines}"
-          ]
+              'core': [
+                  // Due to historical reasons, this is the artifact containing the Gradle metadata,
+                  // see https://twitter.com/benasher44/status/1208944677778534401.
```

Comment:
will the last `.` break the url on hyperlinks?

Reply:
It shouldn't, I can click on it in the IDE.